src/flow-control/log-timings.js

```diff
@@ -0,0 +1,57 @@
+const Rx = require('rxjs');
+const _ = require('lodash');
+const { bufferCount, take } = require('rxjs/operators');
+const BaseHandler = require('./base-handler');
+
+module.exports = class LogTimings extends BaseHandler {
+    printExitInfoTimingTable(exitInfos) {
+        const exitInfoTable = _(exitInfos)
+            .map(({ command, timings, index, killed, exitCode }) => {
+                const readableDurationMs = (timings.endDate - timings.startDate).toLocaleString();
+                return {
+                    'call-index': index,
+                    name: command.name,
+                    duration: `~${readableDurationMs}ms`,
+                    'exit-code': exitCode,
```

Comment:
For some slightly added niceness 😊 

```suggestion
                    'exit code': exitCode,
```